beekeeper-api/src/test/java/com/expediagroup/beekeeper/api/util/DummyHousekeepingMetadataGenerator.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (C) 2019-2021 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.api.util;
+
+
+import static com.expediagroup.beekeeper.core.model.HousekeepingStatus.SCHEDULED;
+import static com.expediagroup.beekeeper.core.model.LifecycleEventType.EXPIRED;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+import com.expediagroup.beekeeper.core.model.HousekeepingMetadata;
+
+public class DummyHousekeepingMetadataGenerator {
+
+  private static final String DEFAULT_DB_NAME = "randomDatabase";
+  private static final String DEFAULT_TABLE_NAME = "randomTable";
+  private static final LocalDateTime CREATION_TIMESTAMP = LocalDateTime.now(ZoneId.of("UTC"));
+  private static final Duration CLEANUP_DELAY = Duration.parse("P3D");
```

Comment:
You don't need to parse. Just do
```suggestion
  private static final Duration CLEANUP_DELAY = Duration.ofDays(3);
```